once-ler.gemspec

```diff
@@ -5,6 +5,7 @@ Gem::Specification.new do |s|
   s.version = '1.1.0'
```

Comment:
If you bump this to 1.1.1, I can make sure it's in the distributed gem as well.

Reply:
@tierra Version bumped. Thank you.